### DISCUSSION TITLE
RTL-SDR V4 dongle support

### DISCUSSION THREAD

**Discussion by ka1ngo** [DSheirer/sdrtrunk#1684] [Q&A]
Greetings Mr. Sheirer
First off thank you for all your hard work on sdrtrunk.
I found it VERY useful during the mass shooting incident here in Maine as i live in the next town over from where the shooter was.
Listening to the Maine State Police on their P25 system kept the infomation up to date. Good to know what's going on.
Anyway i would like to know if support for the new RTL-SDR V4 dongle is in the works? I have it working on GQRX under Linux Mint
20.3 but sdrtrunk didn't work. If you have a fix how do i impliment it?
Again thank you for all your hard work.
Thanks for taking the time to read and respong.

Best regards
Fulton Hunt

**Comment by DSheirer** [comment]
I've added support for v4 tuners.  It will be available in the next release version
